apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -120,6 +120,11 @@ public abstract partial class BigtableClient
                 appProfileId,
                 underlyingClientSettings: null);
 
+        /// <summary>
+        /// Gets the value which specifies routing for replication. If null, the "default" application profile will be used.
```

Comment:
Possibly remove the quotes around default? Otherwise it might be confused for a string which is returned if the profile ID isn't set.

Reply:
Actually, I think I'll just end this sentence (and the others) with "...by the server." That should make it pretty clear.